### PR TITLE
Remove use of six.integer_types/string_types/text_type

### PR DESCRIPTION
**Pull request by foolip**
These are constants corresponding to just int/str/str in Python 3:
https://six.readthedocs.io/#constants

This is simple search-replace with the exception of
resources/test/conftest.py, where element.text is already a string. If
it were None or bytes, then JSON parsing would fail anyway.

In service-workers/cache-storage/resources/vary.py, the reason why
str(cookie_vary) works was non-obvious. Correct the documentation in
tools/wptserve/wptserve/request.py to show that the cookie_vary value
would be a CookieValue, not a binary string.

Since no use of six remains in tools/wptserve/, it's removed as a 
dependency in wptserve's setup.py.

Part of https://github.com/web-platform-tests/wpt/issues/28776.

### CODE REVIEW

File: tools/wptserve/setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 PACKAGE_VERSION = '3.0'
-deps = ["six>=1.13.0", "h2>=3.0.1"]
```
Comment (foolip) [inline tools/wptserve/setup.py:4]: I'm not sure if any setup.py files are actually used, but it seemed best to make this change. Do you know, @jgraham?

**Comment by jgraham** [comment]
I was about to ask if you understood the WebDriver failures, but now you pushed a new version (and of course there's no good interdiff) :)

File: webdriver/tests/new_session/response.py [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
-    assert isinstance(value["sessionId"], string_types)
+    assert isinstance(value["sessionId"], str)
     uuid.UUID(hex=value["sessionId"])
 
 
 @pytest.mark.parametrize("capability, type", [
-    ("browserName", string_types),
-    ("browserVersion", string_types),
-    ("platformName", string_types),
+    ("browserName", str),
```
Comment (foolip) [inline webdriver/tests/new_session/response.py:15]: This changes the names of some tests, and explains some of the noise in https://wpt.fyi/results/webdriver/tests/new_session/response.py?diff&filter=ADC&run_id=5736693317500928&run_id=5762886511099904

The new names are better.

**Comment by foolip** [comment]
@jgraham I didn't change anything, I just wanted to rebase on top of other six changes to ensure no new lint failures. I just commented on part of the webdriver diffs.

Overall, any times I've touched webdriver/ recently the claimed differences have been large. There seems to be a lot of flakiness, have you seen the same? To be expected?

File: tools/wptserve/setup.py (hunk 1 above)
Comment (foolip) [reply tools/wptserve/setup.py:4]: OK, so I'll leave it in.

**Comment by foolip** [comment]
Understanding the CI failures. wpt-chrome-dev-stability and wpt-firefox-nightly-stability timed out in repetition 5 / 10 because too many tests (171) are affected.

There are a lot of apparent changes in the webdriver test results, but there could be flakiness. Combining the two set of before/after runs to look for consistent regressions:
https://wpt.fyi/results/webdriver/tests?q=seq%28status%3Apass%20status%3A%21pass%20status%3Apass%20status%3A%21pass%29&run_id=5768588818382848&run_id=5719745376550912&run_id=5076726092660736&run_id=5716229543165952

Yes, I did break stuff! "FAIL message: NameError: name 'six' is not defined"

I'll iterate until there are no unexplained regressions and the request review.

**Comment by foolip** [comment]
Care must be taken when merging this and https://github.com/web-platform-tests/wpt/pull/28757. Both change `webdriver/tests/support/asserts.py`, and the second to land also needs to remove the `import six` statement. But I don't know which will land first yet.

**Comment by foolip** [comment]
OK, this has now been rebased and the issue in https://github.com/web-platform-tests/wpt/pull/28789#issuecomment-832624636 sorted. Ready for final review.

**Review by jgraham — APPROVED** [review]

**Pull request merged**